src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,21 +2,24 @@
 
 public class DefaultCountingOutRhymer {
 
+	public static final int check = -1;
+	public static final int full = 11;
```

Comment:
Literal full=12 should use constant TABLE_SIZE-1 (use capital letters for constants).